OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -76,6 +76,9 @@ public class ProductionQueue : IResolveOrder, ITick, ITechTreeElement, INotifyOw
 		readonly IEnumerable<ActorInfo> allProducibles;
 		readonly IEnumerable<ActorInfo> buildableProducibles;
 
+		List<Production> productionTraits;
+		List<Production> enabledProductionTraits;
```

Comment:
I don't like the look how how this is used.  It would be cleaner and more conventional to use `productionTraits.Where(not disabled)` instead.